Makefile

```diff
@@ -75,7 +78,11 @@ gobuild:
 
 .PHONY: docker
 docker:
-	@[ ! -z "$(PIPELINE)" ] || docker build --build-arg=GOPROXY=${GOPROXY} --file build/Dockerfile -t "$(BUILD_IMAGE)" .
+	@[ ! -z "$(PIPELINE)" ] || docker build --build-arg=GOPROXY=${GOPROXY} --build-arg=TARGETARCH=$(GOARCH) --file build/Dockerfile -t "$(BUILD_IMAGE)" .
```

Comment:
Is there anything still using this target? Should they still be using this target? If all users should move to `dockerx`, then we can just replace this target with the new one (ie, the contents of `dockerx` goes into this).

Reply:
e2e testing still use this，the Dockerfile could be built by docker directly.

moving to `dockerx` could be next stage.
for testing, `--push` need to be `--load`